src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -24,23 +26,23 @@
     public static Person[] getSamplePersons() {
         return new Person[] {
             new Person(new Nric("S1234567A"), new Name("Alex Yeoh"), new Phone("87438807"),
-                    new Email("alexyeoh@example.com"), new Address("Blk 30 Geylang Street 29, #06-40"),
-                    getTagSet("friends")),
+                        new Email("alexyeoh@example.com"), new Address("Blk 30 Geylang Street 29, #06-40"),
```

Comment:
Why was the formatting changed here?
Same as above comment.
This comment applies for lines 29 - 45.